react-forms/src/TimePicker.tsx

```diff
@@ -32,11 +34,23 @@ export default function DatePicker<T>(props: {
   Disabled?: boolean;
   Feedback?: string;
   Step?: number;
+  Help?: string|JSX.Element;
 }) {
+  const [guid] = React.useState<string>(CreateGuid());
+  const [showHelp, setShowHelp] = React.useState<boolean>(false);
+
   return (
     <div className="form-group">
-      {(props.Label !== "") ?
-      <label>{props.Label == null ? props.Field : props.Label}</label> : null}
+      {(props.Help != null || props.Label !== "") ?
+      <label>{props.Label == null ? props.Field : props.Label}
```

Comment:
I know you didn't update the label element here but <label>{props.Label ?? props.Field}</label>is 'cleaner'.